banzai/bias.py

```diff
@@ -31,9 +31,9 @@ def calibration_type(self):
         return 'bias'
 
     def apply_master_calibration(self, image, master_calibration_image):
-        image -= master_calibration_image.bias_level
+        image -= master_calibration_image.bias_level * image.n_sub_exposures
         image.meta['BIASLVL'] = master_calibration_image.bias_level, 'Bias level that was removed after overscan'
```

Comment:
Should the BIASLVL header be changed to reflect the multiplicative factor of n_sub_exposures?

Reply:
Also, regardless of n_sub_exposures why are we subtracting both bias.level and bias.image, where in line 19, amke_master_calibration, master_iamge is the apparently not level-subtracted?